itou/openid_connect/inclusion_connect/views.py

```diff
@@ -270,7 +270,12 @@ def inclusion_connect_logout(request):
     complete_url = f"{constants.INCLUSION_CONNECT_ENDPOINT_LOGOUT}?{urlencode(params)}"
     # Logout user from IC with HTTPX to benefit from respx in tests
     # and to handle post logout redirection more easily.
-    response = httpx.get(complete_url)
-    if response.status_code != 200:
-        logger.error("Error during IC logout. Status code: %s", response.status_code)
+    # It's okay if this fails... We don't want to crash on our side.
```

Comment:
Why is it okay?

Reply:
Si inclusion connect est down par exemple, ou s'il timeout.
On ne veut pas bloquer la déconnexion de l'utilisateur sur les emploi dans ce cas là.

Reply:
Mon commentaire GitHub voulait dire que le commentaire du code pouvait être précisé. Au lieu de dire “peu importe si ça crash”, on pourrait dire “l’indisponibilité d’IC ne doit pas bloquer la déconnexion des emplois”.